### ISSUE TITLE
Year-Wise tab should have their own url

### ISSUE THREAD

**Issue by TheFlash98** [nihal111/WnCC#108]
If we click on a certain year tab in events, the url remains the same http://wncc-iitb.org/events/
It would be better if the link is something like http://wncc-iitb.org/events/year?=2016 or something. I remember someone did it before.

**Comment by Owl-A** [comment]
Are you sure? I don't think this is correct in practice. 😕

**Comment by grubdragon** [comment]
Events are pretty less. Overkill lag raha hai :eyes:

**Comment by nihal111** [comment]
I think @ydidwania had achieved this before. Can you find out what broke it and fix it? :slightly_smiling_face:

**Comment by grubdragon** [comment]
Right now we are loading everything(all events) and tabbing them

**Comment by Owl-A** [comment]
I agree with grub, @nihal isn't that done for different pages :confused:

**Comment by nihal111** [comment]
Yeah, I agree it's not exactly the "right" way. But to do that with different pages, you need to have a backend that understands the GET request parameters. Since we are using Jekyll, if we want to do this, it'd have to be the Javascript way.

About whether or not we need it, we do have a good number of tabs. It might be a good idea to have a different link for each one of them to ease sharing. Kind of like how the part after the `#` in `https://github.com/nihal111/WnCC/issues/108#issuecomment-308243560`  takes you to that element.

**Comment by grubdragon** [comment]
I guess # approach won't work here. We aren't exactly using links to tab events.
